docs/operating/cdc.md

```diff
@@ -180,3 +180,27 @@ limitations for CDC processing:
 
 Transactions committed after version `0.16.29` are fully compatible with CDC and do not require
 the `history` flag.
+
+
+## Using CDC in Production
+
+### High Availability
```

Comment:
We should add a section header indicating that the following details apply specifically to the AMQP 0.9.1 / RabbitMQ CDC connector, as we will support others.

I don't mind being direct and just saying "RabbitMQ" instead of "AMQP server", but let's use the same names here and [operating/cdc](https://docs.tigerbeetle.com/operating/cdc/).